src/base/system_parameter.c

```diff
@@ -4830,14 +4832,15 @@ static SYSPRM_PARAM prm_Def[] = {
    (char *) NULL,
    (DUP_PRM_FUNC) NULL,
    (DUP_PRM_FUNC) NULL},
-  {PRM_ID_OPTIMIZER_RESERVE_01,
-   PRM_NAME_OPTIMIZER_RESERVE_01,
-   (PRM_FOR_CLIENT | PRM_USER_CHANGE | PRM_HIDDEN),
-   PRM_BOOLEAN,
-   &prm_optimizer_reserve_01_flag,
-   (void *) &prm_optimizer_reserve_01_default,
-   (void *) &PRM_OPTIMIZER_RESERVE_01,
-   (void *) NULL, (void *) NULL,
+  {PRM_ID_MAX_HASH_LIST_SCAN_SIZE,
+   PRM_NAME_MAX_HASH_LIST_SCAN_SIZE,
+   (PRM_FOR_SERVER | PRM_TEST_CHANGE | PRM_SIZE_UNIT),
+   PRM_BIGINT,
+   &prm_max_hash_list_scan_size_flag,
+   (void *) &prm_max_hash_list_scan_size_default,
+   (void *) &PRM_MAX_HASH_LIST_SCAN_SIZE,
+   (void *) &prm_max_hash_list_scan_size_upper,
+   (void *) &prm_max_hash_list_scan_size_lower,
```

Comment:
I can't find where this parameter used. maybe it is omitted?

Reply:
it is in src/query/scan_manager.c
'''
check_hash_list_scan (LLIST_SCAN_ID * llsidp, int *val_cnt, int hash_list_scan_yn)
{
  int build_cnt;
  regu_variable_list_node *build, *probe;
  DB_TYPE vtype1, vtype2;
  UINT64 mem_limit = prm_get_bigint_value (PRM_ID_MAX_HASH_LIST_SCAN_SIZE);
'''